CHANGELOG.md

```diff
@@ -1,19 +1,16 @@
-# Release 0.16.0-dev
-
-### New features
-
-### Improvements
-
-### Breaking changes
+# Release 0.18.0

```

Comment:
We release the v0.18.0 of every plugin along with PennyLane.

Reply:
Oh, is this how we do it? I guess we're fine with skipping version numbers then?

Reply:
Yes, this is partly for our own sanity 😆 

- Easier to keep track of plugin releases if new releases always match the PL release version number
- We can get away with this for now because semantic versioning does not apply < 1.0
- Once we are > 1.0, it should be a lot easier as we will not be introducing breaking changes until a major version increment